app/index.html

```diff
@@ -28,7 +28,7 @@
 <link rel="icon" href="img/icons/512.png" sizes="512x512">
 <link rel="icon" href="img/icon.svg" sizes="any" type="image/svg+xml">
 <title>Project Link</title>
-<script defer src="js/initapp.js"></script>
+<script src="js/alameda.js" data-main="js/bootstrap"></script>
```

Comment:
Can you give me a little more details about `independent chunks` of js? I'm not sure I'm following...

About `defer` - yeah, I think I'll return `defer` here.